source/playgrounds.txt

```diff
@@ -121,6 +121,13 @@ panel, the |vsce| *Overview* page, or the |vscode| Command Pallette.
 
       .. include:: /includes/steps/open-new-playground.rst
 
+.. note::
+
+   You can also open a new playground to test and export queries with the
```

Comment:
Could we also add this note at the top of the "Export to Language" page? That feature now exclusively requires our Copilot extension.

Reply:
Done!